### PR TITLE
Rings Cheatsheet version 20210301 added to Library section

### PR DESCRIPTION
**Pull request by baro77**
Following Addresses' one, I'm PRing a cheatsheet about Ring Signatures into Library section of the site.
I hope it's worth your attention,
baro

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -786,6 +786,9 @@ library:
   moneroaddressescheatsheet20201206: "Monero Addresses Cheatsheet (version 20201206)"
   moneroaddressescheatsheet20201206p: >
     Between the ‘big picture’ and the mathematical details, a single-page recap of various keys, addresses, scopes (private/public, spend/view, on-chain/off-chain, payer’s/payee’s) and their relations.
+  ringscheatsheet20210301: "Rings Cheatsheet (version 20210301)"
```
Comment (erciccione) [inline _i18n/en.yml:789]: I would not put the version. Just for readability, not a hard request

**Review by erciccione — COMMENTED** [review]
I'm asking folks from the MRL to take a look :)

File: _i18n/en.yml (hunk 1 above)
Comment (baro77) [reply _i18n/en.yml:789]: yep I agree, at least until next version will appear (if ever ;) )... so I would delete version from Addresses Cheatsheet as well...
By the way, in your opinion, what about making a "Cheatsheets" area between "Books" and "Magazines"? If you like it and it was my duty as proponent, can you provide some hints? (I'm worried to make a mess acting on site structure)

Comment (erciccione) [reply _i18n/en.yml:789]: > until next version will appear

I'm not sure would be necessary even in that case. You put the version number inside the PDFs, i would say that's enough.

> I would delete version from Addresses Cheatsheet as well

yes, i'm taking care of that right now.

> what about making a "Cheatsheets" area between "Books" and "Magazines"?

Yes that's a good idea. I'll take care of that too after this PR is merged.

**Review by UkoeHB — APPROVED** [review]

**Comment by erciccione** [comment]
@baro77 your second commit is partially redundant with your first commit. I think would be just easier to remove it. It's not a problem to have a second PR :)

**Comment by baro77** [comment]
>  I think would be just easier to remove it.

@erciccione uhmm ok I could "git reset" .... but in that case I would end up again with version in Rings Cheatsheet (I have seen #1497 removes the version only from Address Cheatsheet...).. is that ok? In that case are you going to take care of it as well?

**Comment by erciccione** [comment]
IMO the easiest way to proceed is:

1. Remove your last commit (`git reset --hard HEAD^`)
2. Remove version number from "Rings Cheatsheet". After doing that, add this change to your first commit (`git commit -a --amend --no-edit`)
3. Now force push the new commit: `git push -f origin RingsCheatsheet`

This will remove the redundant commit and the version number from the Rings Cheatsheet, but without creating a new commit. #1497 will remove the versioning number from "Monero Addresses Cheatsheet", so that both will have no version number.

**Comment by baro77** [comment]
> IMO the easiest way to proceed is:

Done! Thanks, I'm never comfortable rewriting history, and I didn't know "--amend" option of commit.. cool (y)

**Review by erciccione — APPROVED** [review]
Thank you. These graphics are very cool and i think are of great help :)

**Pull request merged**